editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Question.tsx

```diff
@@ -76,25 +77,27 @@ const Question: React.FC<Props> = React.memo((props) => {
           },
         )}
       >
-        <Link
-          href={href}
-          prefetch={false}
-          onContextMenu={handleContext}
-          ref={drag}
-        >
-          {props.data?.img && (
-            <Thumbnail
-              imageSource={props.data?.img}
-              imageAltText={props.data?.text}
-            />
+        <Box>
```

Comment:
Wrapping this in the box ensures that the `isClone > div` selector is now properly targeted, matching how checkboxes are already set up.